<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="dropdown_comports">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_debugoutput">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_autobitratematching">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto bit-rate matching</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_write_microkernel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Write control program transfer (micro-kernel)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_linesizeinquiry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Line size inquiry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_device">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_clockmode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select clock mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_baudrate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select baudrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>102</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Perform configuration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_write_main_kernel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Main program transfer (main-kernel)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_boot_program_status_inquiry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Boot program status inquiry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_upload_page">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User MAT blank check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh_ports">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh ports</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
